src/sdk/modules/index.ts

```diff
@@ -1,14 +1,14 @@
-import { OwnableExecutorModule } from "./executors/OwnableExecutor.js"
+// import { OwnableExecutorModule } from "./executors/OwnableExecutor.js"
 import { K1ValidatorModule } from "./validators/K1ValidatorModule.js"
-// import { OwnableValidator } from "./validators/OwnableValidator.js"
+import { OwnableValidator } from "./validators/OwnableValidator.js"
 import { ValidationModule } from "./validators/ValidationModule.js"
 
 export * from "./utils/Types.js"
 export * from "./utils/Constants.js"
 export * from "./utils/Helper.js"
 export * from "./interfaces/IValidationModule.js"
 
-export const createOwnableExecutorModule = OwnableExecutorModule.create
+// export const createOwnableExecutorModule = OwnableExecutorModule.create
```

Comment:
Let's stop doing these aliases here. Bit confusing doing it in a different file to where they actually lives. Shall we move each alias in to the relevant file and export them as the default?